packages/block-library/src/gallery/index.js

```diff
@@ -17,12 +17,12 @@ import { G, Path, SVG } from '@wordpress/components';
  */
 import { default as edit, defaultColumnsNumber, pickRelevantMediaFiles } from './edit';
 
+// Attributes are to be saved as JSON in the block boundary for easier
+// server-side access.
 const blockAttributes = {
 	images: {
 		type: 'array',
 		default: [],
-		source: 'query',
-		selector: 'ul.wp-block-gallery .blocks-gallery-item',
 		query: {
```

Comment:
Should we remove all this query data specifying each property of the object? It looks like it is not used at all now that the source is not a query. If we keep them it may give the idea that it is some "way" to validate the properties of the image object but that is not the case, if the object contains other properties they are passed to the block anyway.